doc/development/developing-sql-dialect/developing_a_dialect.md

```diff
@@ -25,25 +25,25 @@ As an example, PostgreSQL handles some of the data types subtly different from E
 Below you can see a layer model of the Virtual Schemas when implemented with the JDBC adapter.
  The layers in the middle &mdash; i.e. everything that deals with translating between the source and Exasol &mdash; are provided in this repository.
 
-    .-----------------------------------------.
-    |  Exasol    |          Exasol            |
-    |   core     |----------------------------|
-    |            |//// Virtual Schema API ////|
-    |------------|----------------------------|
-    |            |       JDBC  Adapter        |   Common JDBC functions
-    |  In this   |----------------------------|
-    | repository |///// SQL Dialect API //////|
-    |            |----------------------------|
-    |            |    SQL Dialect Adapter     |   Even out specifics of the source database
-    |------------|----------------------------|
-    |            |///////// JDBC API /////////|
-    |            |----------------------------|
-    |            |  PostgresSQL JDBC Driver   |   JDBC compliant access to payload and metadata
-    |  External  |----------------------------|
-    |            |// PostgresSQL Native API //|
-    |            |----------------------------|
-    |            |         PostgreSQL         |   External data source
-    '-----------------------------------------'
+    .-------------------------------------------------.
+    |  Exasol            |          Exasol            |
+    |   core             |----------------------------|
+    |                    |//// Virtual Schema API ////|
+    |--------------------|----------------------------|
+    | In vs-common-jdbc  |       JDBC  Adapter        |   Common JDBC functions
+    |   repository       |----------------------------|
+    |                    |///// SQL Dialect API //////|
+    |                    |----------------------------|
+    | In this repository |    SQL Dialect Adapter     |   Even out specifics of the source database
+    |--------------------|----------------------------|
+    |                    |///////// JDBC API /////////|
+    |                    |----------------------------|
+    |                    |  PostgresSQL JDBC Driver   |   JDBC compliant access to payload and metadata
+    |  External          |----------------------------|
+    |                    |// PostgresSQL Native API //|
+    |                    |----------------------------|
+    |                    |         PostgreSQL         |   External data source
+    '-------------------------------------------------'
```

Comment:
I would remove the current PlantUML diagrams, or change them to something more "workflow" related. Take a look at the SpringMVC workflow for example, in https://tutorialspedia.com/wp-content/uploads/2019/09/Spring-MVC-Flow-Diagram.png and https://java2blog.com/wp-content/uploads/2017/07/SpringMVCTuto-1.png

We could also add a sequence diagram at some point but I think with this should be enough for now.

Reply:
We have a separate ticket for diagrams